### DISCUSSION TITLE
Asset/Plugin update through editor

### DISCUSSION THREAD

**Discussion by GINZUcz** [godotengine/godot-proposals#7818] [Editor]
### Describe the project you are working on

- Pong game in modern way

### Describe the problem or limitation you are having in your project

I see updated assets but i cant update it just reinstall

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

- Add update button and add updated files

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

-

### If this enhancement will not be used often, can it be worked around with a few lines of script?

no

### Is there a reason why this should be core and not an add-on in the asset library?

- I think this will make godot much easier and clearer

**Comment by Calinou** [comment]
Since plugins don't install a manifest that points to a location to check for updates, this is currently not technically feasible (as Godot has no idea where you downloaded the plugin from).
